README.md

```diff
@@ -1,17 +1,21 @@
-# 🥞 syrupy
-[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
+# syrupy
 
-[![Maturity badge - level 1](https://img.shields.io/badge/Maturity-Level%201%20--%20New%20Project-yellow.svg)](https://github.com/tophat/getting-started/blob/master/scorecard.md)
+<span><img align="right" width="200" height="200" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo"></span>
 
 > _/ˈsirəpē/_
 
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
```

Comment:
Need to add a space between badges. If you put them on one line, it'll preserve the single whitespace.

Reply:
Not sure what you mean? You want em on one line?

Reply:
oh nvm, ignore previous comment

Reply:
github markdown diff preview is broken